docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
+
+A how to guide for accessing and running applications on a kubernetes cluster.
+
+### Basic Setup
+The kubernetes cluster is accessed via the ```kubectl`` command.  Full details of using the ```kubectl``` command are described [elsewhere](docs/cli.md).  Make sure that ```kubectl``` is in your path by running
+
+```sh
+which kubectl
+```
+
+If that command returns a path to the binary, you are ready to go.  If it doesn't, you need to update your PATH to include the appropriate directory from the Kubernetes directory.
+
+### Commands
+#### run
+```sh
+kubectl run <pod-name> --image=nginx [--replicas=N] [--labels=<key>=<value>,...][--export [--external]]
```

Comment:
We should probably call out that namespace is set by the context in its own section (as part of the readme): "namespace defaults to Blah, change with the namespace command"


> On Jan 6, 2015, at 11:17 PM, Tim Hockin <notifications@github.com> wrote:
> 
> In docs/cli-usage.md:
> 
> > +
> > +A how to guide for accessing and running applications on a kubernetes cluster.
> > +
> > +### Basic Setup
> > +The kubernetes cluster is accessed via the ```kubectl`` command.  Full details of using the ```kubectl``` command are described [elsewhere](docs/cli.md).  Make sure that ```kubectl``` is in your path by running
> > +
> > +```sh
> > +which kubectl
> > +```
> > +
> > +If that command returns a path to the binary, you are ready to go.  If it doesn't, you need to update your PATH to include the appropriate directory from the Kubernetes directory.
> > +
> > +### Commands
> > +#### run
> > +```sh
> > +kubectl run <pod-name> --image=nginx [--replicas=N] [--labels=<key>=<value>,...][--export [--external]]
> probably needs to take a namespace arg, too
> 
> —
> Reply to this email directly or view it on GitHub.
>